src/fitnesse/components/PluginsClassLoader.java

```diff
@@ -2,45 +2,54 @@
 
 import java.io.File;
 import java.lang.reflect.Method;
+import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.URLClassLoader;
+import java.util.ArrayList;
+import java.util.List;
 
 
+/**
+ * Update the current thread class path with jars foundin a "plugins" directory.
+ */
 public class PluginsClassLoader {
 
-  private File pluginsDirectory;
+  public static ClassLoader loadPlugins(String rootPath) throws Exception {
```

Comment:
This method should probably be renamed to reflect the fact that it doesn't actually load anything, but gets a class loader.

Reply:
I'm fine with this method. It loads them into a ClassLoader.